packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -3,22 +3,24 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-accordion-border-color: [Deprecated] Use `--calcite-accordion-item-border-color`. Specifies the component's border color.
- * @prop --calcite-accordion-item-background-color: Specifies the component's background color.
- * @prop --calcite-accordion-item-border-color: Specifies the component's border color.
- * @prop --calcite-accordion-item-icon-color-end: Specifies the component's `iconEnd` color. Falls back to `--calcite-accordion-item-icon-color` or current color.
- * @prop --calcite-accordion-item-icon-color-start: Specifies the component's `iconStart` color. Falls back to `--calcite-accordion-item-icon-color` or current color.
+ * @prop --calcite-accordion-border-color: Specifies the component's border color.
```

Comment:
@alisonailea Can we add e2e tests, storybook updates, and html demo page updates for these new token variables?